halo1/model.py

```diff
@@ -0,0 +1,76 @@
+import bpy
+
+from reclaimer.hek.defs.mode import mode_def
+from reclaimer.hek.defs.mod2 import mod2_def
+from reclaimer.model.jms import read_jms
+from reclaimer.model.model_decompilation import extract_model
+
+from ..scene.shapes import create_sphere
+
+def read_halo1model(filepath):
+	'''Takes a halo1 model file and turns it into a jms object.'''
+
+	# TODO: Use a tag handler to see if these files actually are what they
+	# say they are. We can get really nasty parsing problems if they aren't.
+
+	# These two model types can be imported the same way because of their
+	# nearly matching structures when built into a python object.
+	if (filepath.lower().endswith('.gbxmodel')
+	or filepath.lower().endswith('.model')):
+		# Load model
+		tag = None
+		if filepath.lower().endswith('.gbxmodel'):
+			tag = mod2_def.build(filepath=filepath)
+		else:
+			tag = mode_def.build(filepath=filepath)
+		#TODO: Get all lod permutations.
+		#Only getting the superhigh perms for now
+		jms = extract_model(tag.data.tagdata, write_jms=False)[0]
+		return jms
+
+	if filepath.lower().endswith('.jms'):
+		# Read jms file into string.
+		jms_string = ""
+		with open(filepath, 'r') as jms_file:
+			jms_string = jms_file.read()
+		# Read Jms data from string.
+		jms = read_jms(jms_string)
+		# Make sure it's a Halo 1 jms
+		if jms.version != "8200":
```

Comment:
Putting a comment here doesn't make this less of a magic number. Let's pull this into a constant